README.md

```diff
@@ -157,6 +157,10 @@ class Page
 
   # default from a singleton value (boolean in this case)
   attribute :published, Boolean, :default => false
+  
+  # default from a singleton value (string in this case),
+  # that doesn't allow a nil value
+  attribute :author, String, :default => 'Mies', :allow_nil => false
```

Comment:
Oh, you're suggesting to use the default if `nil` is passed in. Hmm. That's a harder sell.